testsuite/MDAnalysisTests/__init__.py

```diff
@@ -150,28 +147,4 @@
     in_dir,
     assert_nowarns,
 )
-from MDAnalysisTests.core.util import make_Universe
-
-def run(*args, **kwargs):
```

Comment:
@kain88-de This function has calls to `loaded_plugins` which comes from `from MDAnalysisTests.plugins` which in turn depend upon use `nose`. What should I do?